package.json

```diff
@@ -15,6 +15,7 @@
     "@react-oauth/google": "^0.4.0",
     "@snowplow/browser-tracker": "^3.1.6",
     "@tippyjs/react": "^4.2.6",
+    "@types/lodash.clonedeep": "^4.5.7",
```

Comment:
Underscore's `clone` method is not actually a deep clone, it is shallow ([stackoverflow thread](https://stackoverflow.com/questions/21003059/how-do-you-clone-an-array-of-objects-using-underscore)), so I am using the method from lodash. Thankfully lodash allows us to install just the single function, not the entire library.